liquibase-standard/src/main/java/liquibase/parser/core/xml/XMLChangeLogSAXHandler.java

```diff
@@ -100,7 +100,8 @@ public void endElement(String uri, String localName, String qName) throws SAXExc
         ParsedNode node = nodeStack.pop();
         try {
             String seenText = this.textStack.pop().toString();
-            if (!"".equals(StringUtil.trimToEmpty(seenText))) {
+            String trimmedString = StringUtil.trimToEmpty(seenText);
+            if (!StringUtil.isEmpty(trimmedString)) {
```

Comment:
Hello @Jeevananthammoorthy ! Can't we use the new "StringUtil.isBlank" here and in some other places instead of trimToEmpty then isEmpty?